gocd/pipelines/symbolicator.yaml

```diff
@@ -0,0 +1,131 @@
+# More information on gocd-flavor YAML can be found here:
+# - https://github.com/tomzo/gocd-yaml-config-plugin#pipeline
+# - https://www.notion.so/sentry/GoCD-New-Service-Quickstart-6d8db7a6964049b3b0e78b8a4b52e25d
+format_version: 10
+pipelines:
+    deploy-symbolicator:
+        environment_variables:
+            GCP_PROJECT: internal-sentry
+            GKE_CLUSTER: zdpwkxst
+            GKE_REGION: us-central1
+            GKE_CLUSTER_ZONE: b
+            GKE_BASTION_ZONE: b
+            # Required for checkruns.
+            GITHUB_TOKEN: "{{SECRET:[devinfra-github][token]}}"
+        group: symbolicator
+        lock_behavior: unlockWhenFinished
+        materials:
+            symbolicator_repo:
+                git: git@github.com:getsentry/symbolicator.git
+                shallow_clone: true
+                branch: master
+                destination: symbolicator
+        stages:
+            - checks:
+                  approval:
+                      type: manual
+                  fetch_materials: true
+                  jobs:
+                      checks:
+                          timeout: 1800
+                          elastic_profile_id: symbolicator
+                          tasks:
+                              - script: |
+                                    /devinfra/scripts/checks/githubactions/checkruns.py \
+                                    getsentry/symbolicator \
+                                    ${GO_REVISION_SYMBOLICATOR_REPO} \
+                                    'Unit Tests' \
+                                    'Integration Tests'
```

Comment:
This should be `Tests` and `Sentry-Symbolicator Tests` by now

Reply:
Ah, good catch!